.github/workflows.src/tests.inc.yml

```diff
@@ -208,6 +216,31 @@
           cp build/postgres/stamp build/postgres/install/
         fi
 
+    # Build Stolon
+
+    - name: Set up Go
```

Comment:
Hmm. How much does this slow down the test run?  I'm thinking this should be a separate workflow like the connection pool tests.

Reply:
For the latest (and the only) successful run, it's 8m15s comparing to recent master 7m55s-ish. The test itself is about 1m20s.